Library/Homebrew/dev-cmd/typecheck.rb

```diff
@@ -100,8 +103,6 @@ def typecheck
           end
         end
 
-        Homebrew.failed = system("git", "diff", "--stat", "--exit-code") if args.fail_if_not_changed?
```

Comment:
Thoughts on using `ofail` and `puts` here for consistency with the `update-*` commands?

Reply:
@Rylan12 I talked to @Bo98 about this in Slack: because this command repeatedly uses `safe_system` it is/was not safe to rely on `Homebrew.failed` here as it's been masking actual CI failures in this command for a while.